### DISCUSSION TITLE
Importing a lot af .aax files, is that possible?

### DISCUSSION THREAD

**Discussion by lukethehawk** [audiamus/AaxAudioConverter#129] [Q&A] [answered]
Hi,
I'm trying to use this tool but I'm facing a problem, when I try to import a lot of files the software remain stuck. Is there a limit on the number of files for single import?
thank you

**Comment by audiamus** [comment, marked as answer]
There is probably a practical limit to the number of files, but a few dozen should be possible. However, when loading the files, via the file open dialog or by drag and drop, each file will be analysed first to retrieve essential meta data, before being displayed. The meta data is then shown in the file list. Therefore loading may take a while as FFmpeg is invoked in the background every time to process the file.

**Reply by lukethehawk** [reply]
thank you, I did the job loading about twenty of them at a time
